### DISCUSSION TITLE
Error deserializing hstore using $rawQuery

### DISCUSSION THREAD

**Discussion by xWayfinder** [prisma/prisma#9221] [Q&A] [answered]
Hi all, in my table I have a column with a hstore value which throws the following error when I try and access it using `$rawQuery` is this expected behaviour?

Invalid `prisma.queryRaw()` invocation:  

Raw query failed. Code: `N/A`. Message: `error deserializing column 0: cannot convert between the Rust type `core::option::Option<alloc::string::String>` and the Postgres type `hstore``"

**Comment by ryands17** [comment, marked as answer]
@ImaginateWayne 👋 

We don't support `hstore` at the moment. It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this.

Is it possible for you to use `jsonb` instead of `hstore` as Prisma supports that fully?

**Reply by xWayfinder** [reply]
Thanks @ryands17 for the swift response! 

I think it'll be possible to convert my table to use `jsonb` instead.

Would you still like me to raise a feature request?

**Reply by ryands17** [reply]
If you'd like to then sure you can 🙂

**Comment by xWayfinder** [comment]
Alternatively for those who can't change their data type to `jsonb` there is a function in Postgres called `hstore_to_jsonb()` which can be used in  `$rawQuery`  the downside I guess is that your ORM is no longer DB agnostic.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
